lib/graphql/field/resolve.rb

```diff
@@ -51,7 +61,11 @@ def initialize(field)
         end
 
         def call(obj, args, ctx)
-          obj.public_send(@field.name)
+          if method_with_arguments?(@field.name, obj, args)
+            obj.public_send(@field.name, args)
```

Comment:
Yes we would, but in this kind of example I use class itself as context. I didn't feel any use case so I did not pass it. But anyhow we may need context.